src/main/webapp/WEB-INF/studentForm.jsp

```diff
@@ -12,15 +12,15 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
     <title>New Student</title>
 </head>
 <body>
     <h3>Add a new student:</h3>
     <%--@elvariable id="studentAdd" type="StudentSystem"--%>
     <form:form method="POST" action="/addstudent" modelAttribute="studentAdd">
-        <div class="form-group">
-            <form:label name="id" path="id">Student Id</form:label>
+
+            <form:label name="id" path="id" >Student Id</form:label>
             <form:input path="id"/><br/>
```

Comment:
Since you're using bootstrap, check how you can have things in "rows" instead of adding br tags everywhere.